src/lib/header-bidding/a9/a9.ts

```diff
@@ -85,8 +85,10 @@ const requestBids = async (
 							);
 							if (isInA9BidResponseWinnerTest) {
 								window.guardian.commercial ??= {};
-								window.guardian.commercial.a9WinningBids =
-									bidResponse;
+								window.guardian.commercial.a9WinningBids = [];
```

Comment:
I think resets the property every time we get a bid response, so we'd be back to square one? We only want to set `a9WinningBids` to a new array if it doesn't already exist. Apologies if I'm missing something.

Reply:
No you're absolutely right I was meant to do it like the object above on line 87!

Reply:
I have added window object screenshots in the description to help